lib/licensee/license.rb

```diff
@@ -157,7 +150,14 @@ def title_regex
         Regexp.new string, 'i'
       end
 
-      parts = [title_regex, key]
+      key_regex = begin
+        string = key.sub('-', '[- ]')
+        string.sub!('.', '\.')
+        string << '(?:\ licen[sc]e)?'
```

Comment:
Do we want an optional `the` prefix like above?

Reply:
I don't think so. Above is actually stripping `the` from the license name; `the` isn't part of the regex created at all. `strip_title` doesn't care about beginning of line, only that line contains a match, so `the` is irrelevant. Possibly it should be made more strict, but I don't know of any bug reports triggered by this logic.